src/proto/FunctionRpc.proto

```diff
@@ -271,8 +271,14 @@ message RpcFunctionMetadata {
   // Function language
   string language = 9;
 
-  // Raw binding info
+  // Raw binding info: string of jsonified binding metadata. One string = one binding
   repeated string raw_bindings = 10;
+
+  // Retry Options: string representation of JObject retry options (maxRetryCount, intervals, etc.)
+  string retry_options = 11;
```

Comment:
define new message for retry options - see https://github.com/Azure/azure-functions-language-worker-protobuf/blob/dev/src/proto/FunctionRpc.proto#L327-L336 for an example that was recently added.